app/client/src/components/pages/Community.Tabs.Monitoring.js

```diff
@@ -1319,23 +1263,33 @@ function PastConditionsTab({ setMonitoringDisplayed }) {
 
             <AccordionList
               title={
-                monitoringYearsRange ? (
-                  <span data-testid="monitoring-accordion-title">
-                    <strong>{displayedLocationsCount.toLocaleString()}</strong>{' '}
-                    of <strong>{totalLocationsCount.toLocaleString()}</strong>{' '}
-                    water monitoring sample locations in the{' '}
-                    <em>{watershed}</em> watershed from{' '}
-                    <strong>{monitoringYearsRange[0]}</strong> to{' '}
-                    <strong>{monitoringYearsRange[1]}</strong>.
-                  </span>
-                ) : (
-                  <span data-testid="monitoring-accordion-title">
-                    <strong>{displayedLocationsCount.toLocaleString()}</strong>{' '}
-                    of <strong>{totalLocationsCount.toLocaleString()}</strong>{' '}
-                    water monitoring sample locations in the{' '}
-                    <em>{watershed}</em> watershed.
-                  </span>
-                )
+                <span data-testid="monitoring-accordion-title">
+                  <strong>{displayedLocationsCount.toLocaleString()}</strong> of{' '}
+                  <strong>{totalLocationsCount.toLocaleString()}</strong> water
+                  monitoring sample{' '}
+                  {totalLocationsCount === 1 ? 'location' : 'locations'}
```

Comment:
```suggestion
                  monitoring sample locations
```

This was totally my fault. I didn't check the code I copied from close enough. Using the singular for location here is gramatically incorrect. It is also incorrect in the location that I copied from as well.

Current:
`1 of 1 water monitor sample location with the selected characteristic in the "some place" watershed from 1999 to 2023.`

Suggestion:
`1 of 1 water monitor sample locations with the selected characteristic in the "some place" watershed from 1999 to 2023.`

I can go ahead and commit this change.

Reply:
I thought that sounded off... thanks for making the fixes.